src/interface/src/app/types/plan.types.ts

```diff
@@ -41,6 +42,7 @@ export interface PlanPreview {
   area_m2: number;
   role?: string;
   permissions?: string[];
+  user?: number;
```

Comment:
if this is a number, should this be `userId?`

Reply:
that's the name of the field on the serializer, Im trying to stay away from remapping and transforms. We can update it later on both sides?